sonar-project.properties

```diff
@@ -0,0 +1,6 @@
+sonar.organization=${env.SONAR_ORGANIZATION}
+sonar.projectKey=${env.SONAR_PROJECT_KEY}
```

Comment:
What is this used for, when these properties are also provided as environment variables within the workflow itself?

Reply:
<img width="1121" alt="Screenshot 2021-10-13 at 10 42 34" src="https://user-images.githubusercontent.com/7045335/137099975-bce9ffde-31f4-4915-8e10-eac19ec065ad.png">

Fails.. Some Sonar properties are required it seems. Probably also a way to pass in a prop as an argument, but not going to pilot/explore this at the moment. Achieve the goal to provide some property values as an env variable.